platforms/hyperledger-fabric/configuration/roles/create/console_assets/templates/orgmsp_console.tpl

```diff
@@ -0,0 +1,33 @@
+{
+    "display_name": "{{ item.name | lower }} MSP",
+    "msp_id": "{{ item.name | lower }}MSP",
+    "type": "msp",
+    "admins": [],
+    "root_certs": [
+        "{{ ca_info.CAChain }}"
+    ],
+    "tls_root_certs": [
+        "{{ ca_info.CAChain }}"
+    ],
+    "fabric_node_ous": {
+        "admin_ou_identifier": {
+            "certificate": "{{ ca_info.CAChain }}",
+            "organizational_unit_identifier": "admin"
+        },
+        "client_ou_identifier": {
+            "certificate": "{{ ca_info.CAChain }}",
+            "organizational_unit_identifier": "client"
+        },
+        "enable": true,
+        "orderer_ou_identifier": {
+            "certificate": "{{ ca_info.CAChain }}",
+            "organizational_unit_identifier": "orderer"
+        },
+        "peer_ou_identifier": {
+            "certificate": "{{ ca_info.CAChain }}",
+            "organizational_unit_identifier": "peer"
+        }
+    },
+    "host_url": "https://supplychainconsole.supplychain-net.hf.demo.aws.blockchaincloudpoc.com",
```

Comment:
Seems to be hardcoded URL?

Reply:
This was for test as I was using single console for all organizations, I have changed it to specific org now, in which case, each org will have their own console